website/docs/docs/on-premises/installation/migrating-from-pact-broker.md

```diff
@@ -0,0 +1,43 @@
+---
+id: migrating
+title: Migrating from Pact Broker
+---
+
+# Migrating from Pact Broker
+
+If you have previously used the open source Pact Broker with a PostgreSQL database it is possible to migrate that data to Pactflow, keeping everything and making it available in Pactflow. The migration process uses the existing Pact Broker database. Database migrations are then performed to add additional tables and data used for Pactflow features.
+
+If there is no existing Pact Broker instance continue to the [Database](https://docs.pactflow.io/docs/on-premises/database) section.
```

Comment:
I'm confused about this line. If there is no existing Pact Broker instance, then they wouldn't be in the "Migrating from Pact Broker" docs, yeah?

Reply:
trying to add clarity for people who are just going down the list in this on-premises -> Installation section
![image](https://user-images.githubusercontent.com/13867336/185295927-6dd5bb80-09fe-42b6-9e7b-3eb61ce5f957.png)
Maybe the wrong section for this doc to be in not sure